### PR TITLE
Better Error Reporting for SAAS Portal

### PR DESCRIPTION
**Pull request by tarzan0820**

### CODE REVIEW

File: saas_server/controllers/main.py [hunk 1]
```diff
@@ -17,10 +17,21 @@
 import logging
 _logger = logging.getLogger(__name__)
 
+def webservice(f):
+    @functools.wraps(f)
```
Comment (yelizariev) [inline saas_server/controllers/main.py:21]: functools is not imported

**Comment by yelizariev** [comment]
Found a bug here after merging. We deferentially need an runbot)

**Comment by saltonmassally** [comment]
Agree with you... will do a PR now to fix

**Comment by yelizariev** [comment]
Don't worry, we are about to fix it ourself

**Comment by saltonmassally** [comment]
cool... sorry about the error

**Pull request merged**